src/python/sdk/examples/trade.py

```diff
@@ -482,8 +482,11 @@ def fund_USDC(account: Account, wholes: int):
 
 
 def setup_market(faucet_client: FaucetClient, viewer: EconiaViewer) -> int:
-    lot_size = 10 ** (8 - 3)  # eAPT has 8 decimals, want 1/1000th granularity
-    tick_size = 10 ** (6 - 3)  # eUSDC has 6 decimals, want 1/1000th granularity
+    # Use econia_sdk.utils.decimals.get_market_parameters_integer for these settings.
+    # The settings here give a size precision of 0.001 eAPT and a price precision of
+    # 0.001 eUSDC given eAPT 8 decimals / eUSDC 6 decimals.
+    lot_size = 100000 
+    tick_size = 1
```

Comment:
Good call here, can we also update min size to 500 to be consistent with https://econia.dev/overview/orders#creating-hypothetical-markets?

Reply:
done: a99b7816d360db01025030f898d3ed907db7bb3c